seatunnel-transforms/seatunnel-transforms-flink/seatunnel-transform-flink-sql/src/main/java/org/apache/seatunnel/flink/transform/Sql.java

```diff
@@ -42,16 +42,26 @@
     private static final String SQL = "sql";
 
     @Override
-    public DataStream<Row> processStream(FlinkEnvironment env, DataStream<Row> dataStream) {
+    public DataStream<Row> processStream(FlinkEnvironment env, DataStream<Row> dataStream) throws Exception {
         StreamTableEnvironment tableEnvironment = env.getStreamTableEnvironment();
-        Table table = tableEnvironment.sqlQuery(sql);
+        Table table = null;
+        try {
+            table = tableEnvironment.sqlQuery(sql);
+        } catch (Exception e) {
+            throw new Exception(sql, e.getCause());
+        }
```

Comment:
In my knowledge, this PR is hope to catch the exception caused by sql, and add the wrong sql in exception message. If so, suggest adding whole error message.
e.g. `"Flink SQL transform execute failed, SQL " + sql`.
And if we only use `e.getCause()` to transport the exception, we may lose the exception message.

Reply:
@ruanwenjun 
Yes,thanks for your advice，you mean for example like this?
`throw new Exception("Flink SQL transform execute failed, SQL："+sql, e);`

Reply:
Yes, or you can change the error message.

Reply:
thank you @ruanwenjun, i'll change it

Reply:
Please review，Thank you